plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TableStatisticsReader.java

```diff
@@ -85,33 +86,38 @@
 
     public static final String APACHE_DATASKETCHES_THETA_V1_NDV_PROPERTY = "ndv";
 
-    private final TypeManager typeManager;
-    private final ConnectorSession session;
-    private final Table icebergTable;
-
-    private TableStatisticsReader(TypeManager typeManager, ConnectorSession session, Table icebergTable)
-    {
-        this.typeManager = typeManager;
-        this.session = session;
-        this.icebergTable = icebergTable;
-    }
+    private TableStatisticsReader()
+    { }
```

Comment:
```suggestion
    private TableStatisticsReader() {}
```

Reply:
The class became utility class now

- i have `Utility class 'TableStatisticsReader' is not 'final' ` warning on the class declaration
- move the constructor declaration as the first entry in the class, since this isn't real constructor anymore.